jwt.el

```diff
@@ -613,13 +613,58 @@ Specifically it checks that TEST-STRING has
            (doco (car maybe-doc)))
       (funcall callback doco :thing full-name))))
 
+(defvar jwt--annotation-claims
```

Comment:
Add some documentation for what should go in here and that adding claims will cause the timestamp annotation to show for that claim too.